themes/_core/js/checkout-payment.js

```diff
@@ -112,6 +112,8 @@ class Payment {
   confirm() {
     var option = this.getSelectedOption();
     if (option) {
+      $('.custom-checkbox').click(false);
+      $('.btn-primary').prop('disabled',true);
```

Comment:
Not sure it's the best idea to disable all `.btn-primary` button when an error can occurred and block all buttons.

Reply:
Would you like <code>$('body#checkout #payment-confirmation .btn-primary').prop('disabled',true);</code> instead?

Reply:
No need to chain ids. But you can use exactly the same behavior as above.

```js
$(this.confirmationSelector + ' button').prop('disabled', true);
```
I think this is the only thing you need for this PR. (but not sure =))

Reply:
I have just tested your solution. I can't reproduce the error 👋 Thank you!

Reply:
Thanks to you for your contribution ;) Don't forget you did all the job!